lib/requests/foods.js

```diff
@@ -5,10 +5,20 @@ const getFoods = () => {
     .catch(error => console.error({ error }))
 }
 
-const deleteFood = (id) => {
+const deleteFood = (id, event, removeFoodRow) => {
   fetch(`https://ivmh-qs-api.herokuapp.com/api/v1/foods/${id}`, {
     method: 'DELETE'
   })
+  .then(response => checkSuccessfulDelete(response, event, removeFoodRow) )
+}
+
+const checkSuccessfulDelete = (response, event, removeFoodRow) => {
+  if (response.ok) {
+    removeFoodRow(event)
```

Comment:
I'm a bit confused as far as 'removeFoodRow' being passed in as an argument. Is it not a function that can be called?

Reply:
It is a function that can be called, and we can pass the function object as an argument. I ran into some issues with importing/exporting functions, so by passing the function into this method, we can call it if the delete request went through.

I didn't want to redefine removeFoodRow in this foods.js file since it's a helper for an event listener.